common/reviews/api/ts-command-line.api.md

```diff
@@ -13,8 +13,10 @@ export class AliasCommandLineAction extends CommandLineAction {
     protected onExecute(): Promise<void>;
     // @internal
     _processParsedData(parserOptions: ICommandLineParserOptions, data: _ICommandLineParserData): void;
+    // Warning: (ae-forgotten-export) The symbol "IRegisterDefinedParametersState" needs to be exported by the entry point index.d.ts
```

Comment:
Fix?

Reply:
Whoops, good catch. I forgot that this was still exported.